tests/sentry/backup/test_imports.py

```diff
@@ -856,6 +868,86 @@ def test_global_import_scoping(self):
             == RegionImportChunk.objects.values("import_uuid").first()
         )
 
+    @mock.patch("sentry.backup.imports.is_split_db", return_value=False)
+    @mock.patch("sentry.backup.imports.reset_models")
+    def test_global_import_initial_model_deletion(self, is_split_db, reset_models):
+        if SiloMode.get_current_mode() != SiloMode.MONOLITH:
+            return
+
+        create_default_projects()
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            tmp_empty_file_path = tmp_dir + "empty_backup.json"
+            with open(tmp_empty_file_path, "w") as tmp_file:
+                json.dump([], tmp_file)
+            with open(tmp_empty_file_path, "rb") as empty_backup_json:
+                with assume_test_silo_mode(SiloMode.CONTROL):
+                    import_in_global_scope(empty_backup_json, printer=NOOP_PRINTER)
+                    # Simulate clearing database if reset_models is called. This is a hacky way
+                    # to get this working for tests, since otherwise db transactions won't work on default db
+                    if reset_models.called:
+                        clear_database()
+
+        for dependency in dependencies():
+            model = get_model(dependency)
+            assert model.objects.count() == 0  # type: ignore
+            with connections[router.db_for_read(model)].cursor() as cursor:  # type: ignore
+                cursor.execute(f"SELECT MAX(id) FROM {model._meta.db_table}")  # type: ignore
+                sequence_number = cursor.fetchone()[0]
+                assert sequence_number == 1 or sequence_number is None
+        """
+        During the setup of a fresh Sentry instance, there are a couple of models that are automatically created.
+        The Sentry org, a Sentry team, and an internal project. During a global import, we want to avoid persisting
+        these default models and start from scratch. These explicit assertions are here just to double check that these
+        models have been wiped.
+        """
+        assert Project.objects.count() == 0
+        assert ProjectKey.objects.count() == 0
+        assert Organization.objects.count() == 0
+        assert OrganizationMember.objects.count() == 0
+        assert Team.objects.count() == 0
+
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            path = Path(tmp_dir).joinpath(f"{self._testMethodName}.json")
+            export_to_file(path, ExportScope.Global)
+            with open(path) as tmp_file:
+                assert tmp_file.read() == "[]"
+
+    def test_user_import_persist_existing_models(self):
```

Comment:
nit: `test_persist_existing_models_in_user_scope`. Same for the next two tests (`config` and `organization`). Makes it easier to grep and select with `pytest -k ...`.